src/Telegram/Endpoints/AvailableMethods.php

```diff
@@ -594,6 +594,41 @@ public function setChatAdministratorCustomTitle(
         return $this->requestJson(__FUNCTION__, array_merge($required, $opt));
     }
 
+    /**
+     * Use this method to ban a channel chat in a supergroup or a channel.
+     * The owner of the chat will not be able to send messages and join live streams on behalf of the chat,
+     * unless it is {@see https://core.telegram.org/bots/api#unbanchatsenderchat unbanned} first.
+     * The bot must be an administrator in the supergroup or channel for this to work and must have
+     * the appropriate administrator rights. Returns True on success.
+     * @see https://core.telegram.org/bots/api#banchatsenderchat
+     * @param  string|int  $chat_id Unique identifier for the target chat or username of the target channel (in the
+     *     format [at]channelusername)
+     * @param  int  $sender_chat_id Unique identifier of the target sender chat
+     * @param  array|null  $opt
+     * @return bool|null
+     */
+    public function banChatSenderChat(string|int $chat_id, int $sender_chat_id, ?array $opt = []): ?bool
+    {
+        $required = compact('chat_id', 'sender_chat_id');
+        return $this->requestJson(__FUNCTION__, array_merge($required, $opt));
+    }
+
+    /**
+     * Use this method to unban a previously banned channel chat in a supergroup or channel.
+     * The bot must be an administrator for this to work and must have the appropriate administrator rights.
+     * Returns True on success.
+     * @see https://core.telegram.org/bots/api#unbanchatsenderchat
+     * @param  string|int  $chat_id Unique identifier for the target chat or username of the target channel (in the
+     *     format [at]channelusername)
+     * @param  int  $sender_chat_id Unique identifier of the target sender chat
+     * @return bool|null
+     */
+    public function unbanChatSenderChat(string|int $chat_id, int $sender_chat_id): ?bool
+    {
+        $required = compact('chat_id', 'sender_chat_id');
+        return $this->requestJson(__FUNCTION__, $required);
```

Comment:
```suggestion
        return $this->requestJson(__FUNCTION__, compact('chat_id', 'sender_chat_id'));
```

$required is not used